content/HowToSetUpSocketCANInterface.md

```diff
@@ -0,0 +1,369 @@
+<img src="../files/img/2018_MistyWest_LogoCombo_FINAL_RGB.png" alt="MistyWest" width="400"/><div style="text-align: right">back to [index](../README.md)</div>
+
+# How To Set Up Socket CAN Interface
```

Comment:
Please edit the header similar to the headers of other pages. We changed it for the Github Pages.

Reply:
Oh I see now, I need a title like::
```
---
title: "How To Set Up Socket CAN Interface"
---
```
and not the logo and dthe link back to the index anymore. I understand that this is for functionality reasons but can we consiiiiiiiiiiiider to leave the logo along with the link in place, too?

Reply:
right! Good catch!